src/main/java/com/epam/ta/reportportal/core/integration/util/property/IntegrationDetailsProperties.java

```diff
@@ -45,7 +46,7 @@
 		return ofNullable(details.get(this.attribute)).map(String::valueOf);
```

Comment:
If here can be types other than 'String', it would be more correct to return 'Object' instead of String representation.
Or 'objectMapper' can be added to the 'getValue' args and any 'Object' will be serialized to the 'String' and then will be deserialized.